common/src/test/java/com/microsoft/identity/common/ropc/PasswordGrantTest.java

```diff
@@ -142,21 +141,21 @@ public void testRopcFailureManagedUserNoPassword() throws IOException {
         try {
             final TokenResult tokenResult = performRopcTokenRequest(SCOPES, credential.userName, null);
 
-            assertEquals(true, tokenResult.getSuccess());
+            assertFalse(tokenResult.getSuccess());
         } catch (ClientException exception) {
             fail("Unexpected exception.");
         }
     }
 
     @Test(expected = IllegalArgumentException.class)
-    @Ignore // this has started to fail for some reason, will need investigation
+    // test that ROPC flow fails if scope is not provided
     public void testRopcFailureManagedUserNoScope() throws IOException {
         final Credential credential = getUserCredentialsForManagedUser();
 
         try {
             final TokenResult tokenResult = performRopcTokenRequest(null, credential.userName, credential.password);
 
-            assertEquals(true, tokenResult.getSuccess());
+            assertFalse(tokenResult.getSuccess());
```

Comment:
Does the code even get this far? Isn't the `IllegalArgumentException` thrown before this?

Reply:
Yes, the exception is thrown before that point and code should not reach that far. I wasn't sure if I should put anything there at all. 

I think the right thing to do might be to replace those `assertFalse` statements with an explicit `fail()` so that if we ever get to that line due to coding errors then the test will fail we can catch those bug. What do you say @iambmelt ?

Reply:
@shahzaibj +1

Reply:
I would apply the same approach to the other blocks

Reply:
Addressed